crates/opencascade-sys/include/wrapper.hxx

```diff
@@ -42,6 +42,12 @@
 #include <gp_Trsf.hxx>
 #include <gp_Vec.hxx>
 
+// Generic template constructor
+template <typename T, typename... Args> std::unique_ptr<T> construct_unique(Args... args) {
+  // return T(args...);
```

Comment:
Is this comment needed?

Reply:
Nope that's just me forgetting to remove it :)